Dockerfile-centos8

```diff
@@ -1,31 +0,0 @@
-FROM         centos:8
```

Comment:
Why are we deleting `CentOS 8`?

Reply:
CentOS 8 is now deprecated and they will continue to support CentOS 7.
https://en.wikipedia.org/wiki/CentOS#End-of-support_schedule

I didn't know why pushing CentOS 8, but when I tried to rebuild the latest docker image the repo was outdated and move to another url.

Reply:
I read about it a little bit, it's very confusing 😕 It looks like the entire CentOS project is going to be deprecated in a few years: https://www.zdnet.com/article/centos-linux-8-is-about-to-die-what-do-you-do-next/

I remember we just [updated the CI to use CentOS 8](https://github.com/seladb/PcapPlusPlus/commit/f749142bdb3c09ff88b968e16612bfe127cea1e8). Should we revert back to CentOS 7? Or maybe support both?

Reply:
We decided to go back to CentOS 7

Reply:
@clementperon maybe instead of deleting it we can just move it to `Legacy`?

Reply:
done